tests/genai/evaluate/test_utils.py

```diff
@@ -278,29 +277,6 @@ def test_convert_to_legacy_eval_raise_for_invalid_json_columns(spark):
         _convert_to_eval_set(df)
 
 
-def test_convert_to_eval_set_evaluation_dataset():
```

Comment:
Shouldn't we add a new fixture for EvaluationDataset to `_ALL_DATA_FIXTURES` if we remove this test case?